src/IonBinaryWriter.ts

```diff
@@ -18,11 +18,13 @@ import {Import} from "./IonImport";
 import {IonType} from "./IonType";
 import {IonTypes} from "./IonTypes";
 import {LocalSymbolTable} from "./IonLocalSymbolTable";
-import {LongInt} from "./IonLongInt";
 import {LowLevelBinaryWriter} from "./IonLowLevelBinaryWriter";
 import {TimestampPrecision,Timestamp} from "./IonTimestamp";
 import {Writeable} from "./IonWriteable";
 import {_sign} from "./util";
+import JSBI from "jsbi";
```

Comment:
It means something different.
```typescript
import JSBI from "jsbi";
```
means that `JSBI` is the `default export` in that module. The naming is irrelevant, I could write:
```typescript
import walnuts from "jsbi";
```
and `walnuts` would be the same value as `JSBI` was in the prior example.

If something is the default export, it can't be imported selectively.
```
import {JSBI} from "jsbi";
```
requires that there be a non-default exported value in `"jsbi"` that is named `JSBI`. Running this results in:
```
TS2305: Module ".../jsbi" has no exported member 'JSBI'.
```